airflow/providers/amazon/aws/secrets/secrets_manager.py

```diff
@@ -42,8 +42,11 @@ class SecretsManagerBackend(BaseSecretsBackend, LoggingMixin):
 
     For example, if secrets prefix is ``airflow/connections/smtp_default``, this would be accessible
     if you provide ``{"connections_prefix": "airflow/connections"}`` and request conn_id ``smtp_default``.
-    And if variables prefix is ``airflow/variables/hello``, this would be accessible
+    If variables prefix is ``airflow/variables/hello``, this would be accessible
     if you provide ``{"variables_prefix": "airflow/variables"}`` and request variable key ``hello``.
+    And if configs_prefix is ``airflow/configs/sql_alchemy_conn``, this would be accessible
+    if you provide ``{"configs_prefix": "airflow/configs"}`` and request variable
```

Comment:
```suggestion
    if you provide ``{"configs_prefix": "airflow/configs"}`` and request config
```

Reply:
Updated in https://github.com/apache/airflow/pull/9645/commits/d8d5b8fed26e6f7e04411a43115b3dc4b7c008c8